Airtable integration docs (#1044)
fusedio/fused-docs: docs/guide/data-input-outputs/import-connection/airtable.mdx

---
id: airtable
title: Airtable Integration
sidebar_label: Airtable
---

# Airtable Integration

> Read and write Airtable records from Fused UDFs using OAuth-managed tokens. No API keys need to be saved or hardcoded — Fused handles the token refresh automatically.

Reviewer: ```suggestion
Read and write Airtable records from Fused UDFs by connecting to your Airtable account in Fused.
```

## Prerequisites

> - A Fused account on a paid plan (execution environment required) and an Airtable account.

Reviewer: ```suggestion
- A Fused account on a paid plan and an Airtable account.
```
